flink-connectors/flink-connector-kafka-base/src/test/java/org/apache/flink/streaming/connectors/kafka/KafkaConsumerTestBase.java

```diff
@@ -633,7 +654,12 @@ public void runStartFromGroupOffsets() throws Exception {
 		readProps.setProperty("auto.offset.reset", "earliest");
 
 		// the committed group offsets should be used as starting points
-		KafkaTestEnvironment.KafkaOffsetHandler kafkaOffsetHandler = kafkaServer.createOffsetHandler(standardProps);
+		Properties offsetHandlerProps = new Properties();
+		offsetHandlerProps.putAll(standardProps);
+		offsetHandlerProps.setProperty("key.deserializer", "org.apache.kafka.common.serialization.ByteArrayDeserializer");
+		offsetHandlerProps.setProperty("value.deserializer", "org.apache.kafka.common.serialization.ByteArrayDeserializer");
```

Comment:
Doesn't it make sense to set these properties once in the `KafkaOffsetHandlerImpl` instead of all locations where the offset handler is being created?